components/SongNameForm/SubmitForm.tsx

```diff
@@ -30,6 +40,21 @@ const SubmitForm = () => {
             value={songName}
           />
         </label>
+        {showRecordedVersion && (
+          <label
+            htmlFor="#recodedVersion"
+            className="mt-6 flex flex-col gap-2 text-[#696969] text-[15px]"
+          >
+            <p>Recorded Version</p>
+            <input
+              id="recodedVersion"
+              type="text"
+              className="!outline-none font-rubik rounded-md border-white border !bg-transparent p-2 max-w-[350px]"
+              onChange={(e) => setRecordedVersion(e.target.value)}
+              value={recordedVersion}
+            />
+          </label>
+        )}
```

Comment:
What's the difference between these 2 sets of components?
- To me, they seem `identical` and could better follow the DRY principle.
- Could you please create a shared component called `<FormInput />` or `<NameInput />` to `reduce the total code` while preserving the same functionality?